specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/disk.json

```diff
@@ -783,7 +783,8 @@
           "type": "string",
           "enum": [
             "Standard_LRS",
-            "Premium_LRS"
+            "Premium_LRS",
+            "StandardSSD_LRS"
```

Comment:
This creates invaliad Swagger:
FATAL: System.InvalidOperationException: Swagger document contains two or more x-ms-enum extensions with the same name 'StorageAccountTypes' and different values: Standard_LRS,Premium_LRS vs. Standard_LRS,Premium_LRS,StandardSSD_LRS

Reply:
I forgot to update 2018-04-01. It is updated now.

Reply:
I forgot to update 2018-04-01 compute.json. It is updated now.